components/domains/autorenewal.tsx

```diff
@@ -398,7 +398,8 @@ const Subscription: FunctionComponent<SubscriptionProps> = ({ groups }) => {
                 !address ||
                 !termsBox ||
                 (needMedadata && emailError) ||
-                !areDomainSelected(selectedDomains)
+                !areDomainSelected(selectedDomains) ||
+                !(callData.length > 0) // Cover the case where there are no domains to subscribe
```

Comment:
!(callData.length > 0) is the same as callData.length === 0. Isn't it cleaner that way ?

Reply:
agree